docs/Components-Toggle.md

```diff
@@ -0,0 +1,179 @@
+
+```tsx
+<Toggle value={value} onValueChange={setValue} inputDetailStyle={{ backgroundColor: "#000" }} />
+```
+
+### `labelPostion`
+
+The `labelPosition` prop determines the position of the label relative to the action component. It can be `left` or `right`. It is `right` by default.
+
+```tsx
+<Toggle value={value} onValueChange={setValue} labelPosition="left" />
+```
+
+### `label`
+
+The `label` prop is a string that is used as the label for the toggle.
+
+```tsx
+<Toggle value={value} onValueChange={setValue} label="Remember Me" />
+```
+
+### `labelTx`
+
+The `labelTx` prop is a key to a string in the `i18n` translation file. It is used as the label for the toggle.
+
+```tsx
+<Toggle value={value} onValueChange={setValue} labelTx="login.rememberUsername" />
+```
+
+### `labelTxOptions`
+
+The `labelTxOptions` prop is an object that is passed to the `i18n` translation function. It is used to pass in values to the translation string.
+
+```tsx
+<Toggle
+  value={value}
+  onValueChange={setValue}
+  labelTx="login.rememberUsername"
+  labelTxOptions={{ username: "john" }}
+/>
+```
+
+### `labelStyle`
+
+The `labelStyle` prop is a `StyleProp<TextStyle>` object that is applied to the label.
+
+```tsx
+<Toggle value={value} onValueChange={setValue} labelStyle={{ color: "#000" }} />
+```
+
+### `LabelTextProps`
+
+The `LabelTextProps` prop is a `TextProps` object (from the [`Text`](./Components-Text.md)) component that is applied to the label.
+
+```tsx
+<Toggle value={value} onValueChange={setValue} LabelTextProps={{ size: "lg" }} />
+```
+
+### `helper`
+
+The `helper` prop is a string that is used as the helper for the toggle.
+
+```tsx
+<Toggle value={value} onValueChange={setValue} helper="Remember Me" />
+```
+
+### `helperTx`
+
+The `helperTx` prop is a key to a string in the `i18n` translation file. It is used as the helper for the toggle.
+
+```tsx
+<Toggle value={value} onValueChange={setValue} helperTx="login.rememberUsername" />
+```
+
+### `helperTxOptions`
+
+The `helperTxOptions` prop is an object that is passed to the `i18n` translation function. It is used to pass in values to the translation string.
+
+```tsx
+<Toggle
+  value={value}
+  onValueChange={setValue}
+  helperTx="login.rememberUsername"
+  helperTxOptions={{ username: "john" }}
+/>
+```
+
+### `HelperTextProps`
+
+The `HelperTextProps` prop is a `TextProps` object (from the [`Text`](./Components-Text.md)) component that is applied to the helper.
+
+```tsx
+<Toggle value={value} onValueChange={setValue} HelperTextProps={{ size: "lg" }} />
+```
+
+### `switchAccessibilityMode`
+
+The `switchAccessibilityMode` is a special prop for the switch variant that adds a text/icon label for on/off states. Options are `text` and `icon`
+
+```tsx
+<Toggle value={value} onValueChange={setValue} variant="switch" switchAccessibilityMode="icon" />
+```
```

Comment:
I recently added another wrapper around the input called `inputWrapperStyle`. Le't document that. Here are the differences: 
![yulolimum-capture-2022-09-28--07-44-10](https://user-images.githubusercontent.com/1775841/192810427-ec904459-3c80-414a-8d1e-6589c9d7dc13.jpg)